src/ddscxx/include/org/eclipse/cyclonedds/topic/qos/TopicQosDelegate.hpp

```diff
@@ -84,6 +85,7 @@ class OMG_DDS_API TopicQosDelegate
     dds::core::policy::TransportPriority      priority_;
     dds::core::policy::Lifespan               lifespan_;
     dds::core::policy::Ownership              ownership_;
+    dds::core::policy::DataRepresentation     datarepresentation_;
```

Comment:
Please also encapsulate this line with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro. (BTW: I see that in this particular case the embedding OMG_DDS_OWNERSHIP_SUPPORT macro was also missing for the "ownership_" attribute).

Reply:
Will do.